lib/DDG/Spice/IndeedJobs.pm

```diff
@@ -18,15 +18,27 @@ attribution github  => ['parker', 'Parker'],
             twitter => ['ourmaninjapan', 'Daniel Davis'],
             web     => ['http://daniemon.com/', 'Daniel Davis'];
 
-triggers any => "///***never trigger***///";
+triggers any => "job", "jobs";
```

Comment:
this should stay as it is. We're using a special internal triggering method. Please see https://github.com/duckduckgo/zeroclickinfo-spice/commit/afa6edb916b3dcb8a9956123e9f4f88ed54fc0f5 for more info